packages/components/src/internal/components/chart/models.ts

```diff
@@ -3,6 +3,7 @@ import { Filter, Query } from '@labkey/api';
 
 import { AppURL } from '../../url/AppURL';
 import { SchemaQuery } from '../../../public/SchemaQuery';
+import {VisualizationGetResponse} from "@labkey/api/dist/labkey/query/Visualization";
```

Comment:
```suggestion
import { VisualizationGetResponse } from "@labkey/api/dist/labkey/query/Visualization";
```

Looks like we need to explicitly export `VisualizationGetResponse` from `@labkey/api`, or fix this import if we are already exporting it.

Reply:
good catch. looks like I might need to bump @labkey/api to export VisualizationGetResponse

Reply:
nvm, it is already there under the `Visualization` namespace